orchestration/dags/data_gcp_dbt/macros/custom_test/not_null_proportion_test.sql

```diff
@@ -0,0 +1,23 @@
+{% test not_null_proportion(model, column_name, where_condition, warn_if) %}
+
+    with
+        validation as (
+            select
+                count(*) as total_records,
+                sum(
```

Comment:
count(CASE WHEN NOT {{ column_name }} THEN 1 END) as matching_records

Reply:
ça marche pas pcq les colonnes testées ne sont pas des bool